Formula/k2cli.rb

```diff
@@ -0,0 +1,17 @@
+class K2cli < Formula
+  desc "CLI for K2"
+  homepage "https://github.com/samsung-cnct/k2cli"
+  url "https://github.com/samsung-cnct/k2cli/releases/download/1.0.5/k2cli_1.0.5_darwin_amd64.tar.gz"
```

Comment:
@dunn - Going chat with my team, i think the version is a misunderstanding I am having with my team. I suppose we can build from source just fine, but out of curiosity is there  a reason we would want to use this approach when the releases already have the precompiled binary? Also, would appreciate advice about testing, as i mentioned, docker is an external resource we expect in order to run the test (that isnt trivial) and this binary heavily relies on docker. If not ill go ahead and do the trivial test.